lug-helper.sh

```diff
@@ -679,11 +682,13 @@ getdirs() {
     # $game_version is set in the version_menu() function
     ############################################################################
     # The game's user directory
-    user_dir="$game_path/$game_version/USER/Client/0"
+    user_dir="$game_path/$game_version/user/client/0"
     # The location within the USER directory to which the game exports keybinds
     keybinds_dir="$user_dir/Controls/Mappings"
-    # Shaders directory
-    shaders_dir="$wine_prefix/$appdata_path"
+    # game data mask
+    game_data_mask="sc-alpha-*"
```

Comment:
I'm thinking we might want a better name for this variable. At first glance, it doesn't really tell us which directory it's going to be referencing. Perhaps `appdata_mask`? Or maybe `appdata_dir_mask`? :thinking: